H5P.BranchedVideo/branched-video.js

```diff
@@ -517,14 +518,14 @@ H5P.BranchedVideo = (function ($) {
           // arrange branch text and time text
           var branchText = nextBranch.getBranchTextHTML();
           if (nextBranch.getType() == 'top'){
-            branchText.style.top = - 9  + 'px';
+            branchText.style.top = 6  + 'px';
```

Comment:
Just wondering (not sure): Is it the best approach to hard-code these values in or do you think it makes sense to use teh spaceBetweenBranches to come up with this as well? Maybe as a percentage or something? I just want to make sure that if we change the value of spaceBetweenBranches, the text will always be centered vertically between the two lines. If it's not an issue, then you can ignore this comment.